util/dogfood.sh

```diff
@@ -0,0 +1 @@
+rm -rf target* && cargo build --lib && cp -R target target_recur && cargo rustc -- -Zextra-plugins=clippy -Ltarget_recur/debug   -Dclippy
```

Comment:
remove target_recur afterwards?

@llogiq had a simpler invocation, does that work for you?